### PR TITLE
adding board acceptance

### PR DESCRIPTION
**Pull request by Amet015**

### CODE REVIEW

File: trello/features/acceptance/BoardAcceptance.feature [hunk 1]
```diff
@@ -0,0 +1,49 @@
+# Created by Alan at 1/27/2020
+Feature: CRUD of Board
```
Comment (Enrique-C) [inline trello/features/acceptance/BoardAcceptance.feature]: Please, review this name

File: trello/features/acceptance/BoardAcceptance.feature [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+  Scenario: Change the name of a board
```
Comment (Enrique-C) [inline trello/features/acceptance/BoardAcceptance.feature]: It would be "Changes ...."

File: trello/features/acceptance/BoardAcceptance.feature [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+    Given Sets a POST request to "https://api.trello.com/1/boards/" and save as "url"
```
Comment (Enrique-C) [inline trello/features/acceptance/BoardAcceptance.feature]: It returns a response or url?

File: trello/features/acceptance/BoardAcceptance.feature [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+      {
+      "name": "GherkinBoard",
+      "desc": "",
+      "descData": null,
+      "closed": false,
+      "idOrganization": null,
+      "idEnterprise": null,
+      "pinned": false
+      }
+    """
```
Comment (Enrique-C) [inline trello/features/acceptance/BoardAcceptance.feature]: I is not necessary

File: trello/features/acceptance/BoardAcceptance.feature [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+    And Sets a PUT request to /boards/"Board.id"
+    And sets the following Path Params
+       """
+        {
+          "id": "{Board.id}"
+          "name":"BoardUpdated"
+        }
+       """
+    Then Sends the Put request
```
Comment (Enrique-C) [inline trello/features/acceptance/BoardAcceptance.feature]: It would be When

File: trello/features/acceptance/BoardAcceptance.feature (hunk 3 above)
Comment (Amet015) [reply trello/features/acceptance/BoardAcceptance.feature]: no, I just want to save the url in that variable

File: trello/features/acceptance/BoardAcceptance.feature [hunk 6]
```diff
@@ -1,49 +1 @@
 # Created by Alan at 1/27/2020
```
Comment (Enrique-C) [inline trello/features/acceptance/BoardAcceptance.feature]: Add your last Name

**Review by Enrique-C — APPROVED** [review]

**Pull request merged**